### DISCUSSION TITLE
Calendar: on blur, typing an invalid value clears the input

### DISCUSSION THREAD

**Discussion by omzy** [primefaces/community#870] [PrimeVue]
We use custom validation on all of our form controls, which is triggered on blur. This allows us to handle invalid input, so for example we allow the user to enter an invalid value and then on blur, we validate that value and show an error message letting the user know what they did wrong.

Unfortunately, I cannot do this on the Calendar control, because it seems to clear the value when I focus away from it (when using `manualInput`).

See demo here: https://stackblitz.com/edit/g1vpgm?file=src%2FApp.vue

In this demo, I have set a max date of "today + 7 days". If I type in a date outside of this constraint and then focus out of the field, it just clears the value. It also does the same thing if I type in a completely invalid date, e.g. "50/50/100"

Ideally, I would prefer if it keeps whatever value has been typed in and allow me to use my on blur handler to mark the field as invalid.

**Comment by MiguelBamberino** [comment]
Good Afternoon,
Is there a response on when/If you'll be able to fix this issue ?

**Reply by trajano** [reply]
This was closed off a while back but I don't think it works based on the follow up comments.
https://github.com/primefaces/primevue/issues/1825

**Comment by omzy** [comment]
Hello, I was just wondering whether this is likely to be fixed?
